compiler/rustc_codegen_llvm/Cargo.toml

```diff
@@ -13,6 +13,7 @@ cstr = "0.2"
 libc = "0.2"
 libloading = "0.7.1"
 measureme = "10.0.0"
+object = { version = "0.28.3", default-features = false, features = ["std", "read_core", "archive", "coff", "elf", "macho", "pe"] }
```

Comment:
I think this should be `0.29` -- it was upgraded recently.

Reply:
Updated and rebased